src/main/java/com/googleinterns/zoomtube/utils/TranscriptLineUtil.java

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.googleinterns.zoomtube.data.TranscriptLine;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+/** Provides methods to create TranscriptLine Entities and TranscriptLine objects. */
+public class TranscriptLineUtil {
+  public static final String XML_URL_TEMPLATE = "http://video.google.com/timedtext?lang=en&v=";
+  public static final String ATTR_START = "start";
+  public static final String ATTR_DURATION = "dur";
+  public static final String TAG_TEXT = "text";
```

Comment:
This is all unrelated to the transcriptline autovalue/entities.  I think it makes more sense for them to be in the servlet where you're actually handling the XML